_source/user-guide/accounts/siem_shared_repository.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: article
+title: Manage the shared SIEM Repository
+permalink: /user-guide/accounts/shared_repository.html
+flags:
+  admin: true
+  logzio-plan: pro
+tags:
+  - security
+  - security-account
+  - accounts
+  - logz.io-security
+  - siem
+contributors:
+  - yberlinger
+---
+
+## About the shared SIEM Repository 
+
+Build, prepare, and test your security content in a single sub account, set it as the  as the shared SIEM Repository, and share the content in the Repository with your other SIEM accounts.
```

Comment:
Sub account might be confusing because it is the name of logging sub accounts and on account management we have a section called "sub accounts" which is not the account we refer to.
The repository account is a security (SIEM - we might need to be consistent with the names? I'm no longer sure what it is) account too.

Reply:
Changed to:
Build, prepare, and test your security content in a single SIEM account, set it as the shared SIEM Repository, and share the content in the Repository with your other SIEM accounts.